Makefile

```diff
@@ -60,7 +60,7 @@ release:
   	  -DBUILD_LAME="${BUILD_LAME}"             \
   	  -DCMAKE_SKIP_RPATH="${NO_RPATH}"     ..; \
       ${BUILD_SYSTEM} lrelease;                             \
-      ${BUILD_SYSTEM} -j ${CPUS};                           \
+      ${BUILD_SYSTEM};                           \
```

Comment:
I was just trying it out. It worked on my local machine but for some reason this caused the build to fail on Travis.